app/src/main/java/com/nutomic/syncthingandroid/activities/FolderActivity.java

```diff
@@ -162,26 +168,39 @@ public void onCreate(Bundle savedInstanceState) {
             // Open keyboard on label view in edit mode.
             binding.label.requestFocus();
             binding.editIgnores.setEnabled(false);
+            setOverrideChangesContainerEnabled(false);
         }
         else {
             // Prepare edit mode.
             binding.id.clearFocus();
             binding.id.setFocusable(false);
             binding.id.setEnabled(false);
             binding.directoryTextView.setEnabled(false);
+
+            // overridable remotes button
+            setOverrideChangesContainerEnabled(
+                getIntent().getBooleanExtra(EXTRA_FOLDER_OVERRIDABLE_CHANGES, false)
+            );
         }
 
         if (savedInstanceState != null){
-            if (savedInstanceState.getBoolean(IS_SHOWING_DELETE_DIALOG)){
+            if (savedInstanceState.getBoolean(IS_SHOW_DELETE_DIALOG)){
                 showDeleteDialog();
             }
         }
 
         if (savedInstanceState != null){
-            if (savedInstanceState.getBoolean(IS_SHOWING_DELETE_DIALOG)){
+            if (savedInstanceState.getBoolean(IS_SHOW_DELETE_DIALOG)){
                 showDeleteDialog();
             }
         }
```

Comment:
Entirely optional, just suggesting it as you are renaming the constant which is also a somewhat unrelated cleanup: You could delete one of the two redundant blobs of code.